content/courses/Ada_For_The_C_Embedded_Developer/chapters/06_Translation.rst

```diff
@@ -134,10 +134,40 @@ And that's all that's necessary. Here's an example of a call to :ada:`Call`:
 Building and Debugging mixed language code
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
+The easiest way to build an application using mixed C / Ada code is to create
+a simple project file for :program:`gprbuild` and specify C as an additional
+language. By default, when using :program:`gprbuild` we only compile Ada source
+files. To compile C code files as well, we use the ``Languages`` entry and
```

Comment:
I suggest "entry" should be "attribute" since "entry" is used elsewhere as a language construct and "attribute" is the actual term in case they look it up in the user guide.